apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -200,7 +203,11 @@ const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, on
 									</ul>
 									<p>
 										{ __( 'Questions?', 'full-site-editing' ) }{ ' ' }
-										<Button isLink href="https://wordpress.com/help/contact" target="_blank">
+										<Button
+											isLink
+											href={ localizeUrl( 'https://wordpress.com/help/contact', null, window.currentUser ) }
```

Comment:
There is a TypeScript error because of this line:
> Argument of type 'null' is not assignable to parameter of type 'string | undefined'.

Then,`window.currentUser` seem to be always `undefined` in this context but we know that a user needs to be logged in to be able to launch their site, right? 

After checking the `useLocalizeUrl` callback arguments at https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L153 the correct function call would be:
```
localizeUrl( 'https://wordpress.com/help/contact', locale, true )
``` 

But `localizeUrl` at https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L104 defaults last argument (` isLoggedIn`) to true so we can just omit that.
```suggestion
											href={ localizeUrl( 'https://wordpress.com/help/contact', locale ) }
```